test/address_test.rb

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+require "yast"
+
+Yast.import "UI"
+
+class DummyClass < Yast::Module
+  def initialize
+    Yast.include self, "network/lan/address.rb"
+  end
+end
+
+describe "NetworkLanAddressInclude" do
+  subject { DummyClass.new }
+
+  describe "#justify_dev_name" do
+
+    it "returns given device name justified by 0's at right" do
+      expect(subject.justify_dev_name("em5p1")).to eq("em00005p00001")
+    end
+
+  end
+
+  describe "wrap_text" do
+    subject(:routines) { DummyClass.new }
+    let(:devices) do
```

Comment:
np: you can use until 100 characters by line (maybe it fits into the same line, I haven't tried it).